dolphin/driver_manager/manager.py

```diff
@@ -0,0 +1,57 @@
+import shlex
+import subprocess
+from oslo_log import log
+from oslo_service import time
+from oslo_utils import uuidutils
+
+LOG = log.getLogger(__name__)
+
+DEVICE_INFO = {
+    'id': uuidutils.generate_uuid(),
+    'name': 'string',
+    'description': 'string',
+    'vendor': 'string',
+    'status': 'available',
+    'total_capacity': 'double',
+    'used_capacity': 'double',
+    'free_capacity': 'double',
+    'manufacturer': 'string',
+    'model': 'string',
+    'firmwareVersion': 'string',
+    'serial_number': 'string',
+    'location': 'string',
+    'created_at': 'string',
+    'updated_at': 'string'
+}
+
+
+class Driver:
+    def __init__(self):
+        self.stderr = None
+        self.stdout = None
+
+    def run_command(self, command):
+        args = shlex.split(command)
+        process = subprocess.Popen(args,
+                                   stdout=subprocess.PIPE,
+                                   stderr=subprocess.PIPE)
+        stdout, stderr = process.communicate()
+        if stderr != '':
+            raise Exception(stderr)
+        else:
+            LOG.info("Stdout: {0}".format(stdout))
+        return stdout, stderr
+
```

Comment:
Is this a temporary implementation? If yes, Please mark/comment accordingly. This seems to be test placeholder